### ISSUE TITLE
Support for group mapping from mulitple (nested?) entries

### ISSUE THREAD

**Issue by col-panic** [nextcloud/user_oidc#799]
We use keycloak and this plugin. We have both realm roles, and nextcloud specific client roles (like `admin` which represents the nextloud admin group). 

Keycloak by default delivers the users realm and client roles like this

```
  "realm_access": {
    "roles": [
      "offline_access",
      "uma_authorization",
      "medical-practitioner",
      "medical-user",
      "user",
      "mpa"
    ]
  },
  "resource_access": {
    "nextcloud": {
      "roles": [
        "admin"
      ]
    }
  }
``` 

So in this case, the groups to apply for nextcloud would be: `[ "offline_access", "uma_authorization", "medical-practitioner", "medical-user", "user", "mpa"]`AND `["admin"]`. At the moment it is only possible to set a single attribute as the source for groups and I am not even sure, if a nested attribute would be accepted.

* So to map only the realm roles, I would have to be able to set `realm_access.roles` as source for `groups` (not sure if this is supported)

But my real feature request is to support a group attribute entry like this `realm_access.roles,resource_access.nextcloud.roles` - this way I would not have to introduce an additional mapper, but nextcloud (with this plugin) could directly use the groups as expected.

**Comment by edward-ly** [comment]
I don't think this feature is in our roadmap, but feel free to submit a pull request if you think it would be valuable and you're up to the task.
